exchange/exchange.go

```diff
@@ -167,12 +169,95 @@ func (e *exchange) HoldAuction(ctx context.Context, bidRequest *openrtb.BidReque
 			}
 			targData.setTargeting(auc, bidRequest.App != nil, bidCategory)
 		}
+
+		if requestExt.Prebid.SupportDeals {
+			dealErrs := applyDealSupport(bidRequest, auc)
+			if len(dealErrs) > 0 {
```

Comment:
Is the length check needed? Go's builtin methods are kind to nil or empty collections, so `errs = append(errs, dealErrs...)` might work just fine.

Reply:
Length check is indeed not needed, removed.